source/mir/ndslice/topology.d

```diff
@@ -323,6 +382,26 @@ version(mir_test) unittest
     static assert(slice._strides.length == 0);
 }
 
+@safe pure nothrow
```

Comment:
Mark all unittests with empty comment. This will push them into the documentation.

Reply:
Done in 5c98712
Apologies for overlooking the detail.